common/test/acceptance/tests/video/test_studio_video_editor.py

```diff
@@ -126,6 +126,27 @@ def test_translations_uploading(self):
         self.assertIn(unicode_text, self.video.captions_text)
         self.assertEqual(self.video.caption_languages.keys(), ['zh', 'uk'])
 
+    def test_save_language_upload_no_transcript(self):
+        """
+        Scenario: Transcript language is not shown in captions if no transcript file is uploaded
+        Given I have created a Video component
+        And I edit the component
+        And I open tab "Advanced"
+        And I add a language "uk" but do not upload an .srt file
+        And I save changes
+        Then when I view the video
```

Comment:
`When I view the video captions/language menu`
`Then I cannot see the language "uk" translation`